fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -732,6 +734,9 @@ public boolean getUseScanBlockCache() {
 
     @VarAttr(name = QUERY_CACHE_ENTRY_MAX_ROWS)
     private long queryCacheEntryMaxRows = 409600;
+
+    @VarAttr(name = IGNORE_COM_FIELD_LIST)
+    private boolean ignoreComFieldList = false;
```

Comment:
Better to make its default value true, then the behavior won't change after upgrading.

Reply:
The meaning of this variable is whether to skip the command. If the value of the variable is false, it means no skiping, which is as expected.